.github/workflows/test.yaml

```diff
@@ -9,24 +9,43 @@ on:
       - bugfix/*
       - feature/*
 jobs:
-  bdist:
-    name: build bdist wheels and test
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-22.04, macos-11, windows-2019]
+  windows-i686:
+    name: build windows wheels and test
+    runs-on: windows-2019
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
           python-version: '3.10'
-      - if: runner.os == 'Linux'
-        uses: docker/setup-qemu-action@v2
+      - name: Build
+        uses: pypa/cibuildwheel@v2.12.0
         with:
-          platforms: all
-      # macOS make is too old
-      - if: runner.os == 'macOS'
-        run: brew install make
+          config-file: "{package}/ci-win32.toml"
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./wheelhouse/*.whl
+  windows-x86_64:
+    name: build windows wheels and test
```

Comment:
perhaps, we could use matrix here. but I don't touch it as it is working.

Reply:
Edit by maintainers is on for this PR, feel free to apply matrix on this change.